WordPress/src/main/res/layout/post_list_author_selection_dropdown.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:foreground="?attr/selectableItemBackground"
+              android:gravity="center"
+              android:orientation="horizontal"
+              android:padding="4dp"
+              tools:ignore="UnusedAttribute">
+    <!-- UnusedAttribute is ignored since foreground was added in API 23, however the impact of the missing foreground
```

Comment:
The background gets set programatically depending on whether the item is selected or not.

I can add a comment to clarify this if you think it will help?